quartical/apps/backup.py

```diff
@@ -41,7 +42,14 @@ def backup():
         args.ms_path,
         columns=args.column,
         index_cols=("TIME",),
-        group_cols=("DATA_DESC_ID",))
+        group_cols=("FIELD_ID","DATA_DESC_ID","SCAN_NUMBER"))
```

Comment:
Also, we may as well set chunks={'row': 125000} here.

Reply:
Do you mean that, if writing to an ms, we need to group columns the same? If so I am not sure how to do that because I don't think xds_to_table accepts a group_cols argument, thought it figures it out automatically based on rowid. Would be happy to make the change if I knew what to do

Reply:
Ah nevermind. Confusing the app with what QC itself does. What I meant is that if we were reading the dataset and merging in the column, we would want to be consistent. But of course, I didn't implement it like that - it just writes the single column. In fact, there is probably something to be tested here as with the changes to `group_cols` I am not sure that we can guarantee that the rows we read in will be in the same order if the data isn't natively time ordered. Just something to think about.